Web/HTML1/MetaGurukul.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+	<form>
+	<fieldset form="">
+		<table>
+		<tr><td align="right"><button type="button">Help</td></tr>
```

Comment:
ACTION: According to the requirement this button should be aligned to the top right of the page but in your case it is coming in the middle because you are aligning within a td tag